3 mukemmel_sayi.py

```diff
@@ -0,0 +1,13 @@
+def mukemmel(sayi):
+    adet = 0;
+    for i in range(1, sayi):
+        if sayi % i == 0:
+            adet += i
+    if adet == sayi:
+        print(sayi, " mukemmel sayidir.")
+    else:
+        print(sayi, " mukemmel sayi degil")
+        return
```

Comment:
`return` satırının yanına bir şey yazmadığınız için fonksiyon her çalıştığında `None` değerini geri dönüyor. Fonksiyon içinde `print` kullandığınız için `return` ifadesini silebilirsiniz.